Dockerfile.asserts

```diff
@@ -0,0 +1,21 @@
+# Build the manager binary
+FROM golang:1.14 as builder
+
+COPY . /go/src/github.com/jaegertracing/jaeger-operator/
+WORKDIR /go/src/github.com/jaegertracing/jaeger-operator
+
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build  -a ./tests/assert-jobs/allinone_testuiconfig.go
```

Comment:
Is this test specific to the allinone?

Reply:
yes, but may be we could reuse in the future, but at this point this is allinone specific.